tsconfig.json

```diff
@@ -1,26 +1,31 @@
 {
-  "compilerOptions": {
-    "target": "es6",
-    "module": "commonjs",
-    "sourceMap": true,
-    "moduleResolution": "node",
-    "allowSyntheticDefaultImports": true,
-    "experimentalDecorators": true,
-    "esModuleInterop": true,
-    "emitDecoratorMetadata": true,
-    "strict": true,
-    "skipLibCheck": true,
-    "typeRoots": [
-      "node_modules/@types",
-      "types"
+    "compilerOptions": {
+        "target": "es6",
+        "module": "commonjs",
+        "sourceMap": true,
+        "moduleResolution": "node",
+        "allowSyntheticDefaultImports": true,
+        "experimentalDecorators": true,
+        "esModuleInterop": true,
+        "emitDecoratorMetadata": true,
+        "strict": true,
+        "skipLibCheck": true,
+        "typeRoots": [
+            "node_modules/@types",
+            "types"
+        ],
+        "strictPropertyInitialization": false
+    },
+    "include": [
+        "src/**/*.ts",
+        "test/**/*.ts",
+        "src/Migrations/20191224091606-create-node.js",
```

Comment:
why this includes are added?

Reply:
Didn't see those, they were probably added when generating Models, should I just remove the duplicate ones, leave to include the Migrations by name or remove the all Migrations inclusions?